### PR TITLE
Updated pr-data.csv for 1 opened PR and 1 moved test

### PR DESCRIPTION
**Pull request by Qi-Cui**
Report 1 opened PR and 3 moved tests.

Moved tests:
- nifi(https://github.com/apache/nifi): test `testReferencedUserWithoutDefiningReferencedAttribute` originally under module `nifi-nar-bundles` was moved to module `nifi-framework-bundle` in commit `e4ec0cb20d63a38c4e8da169b0bbd26d0317fb90`.

### CODE REVIEW

**Review by darko-marinov — CHANGES_REQUESTED** [review]
What evidence (e.g., log files) do you have that some tests are `ID` in some commits, e.g., `testReferencedUserWithoutDefiningReferencedAttribute` in `e4ec0cb20d63a38c4e8da169b0bbd26d0317fb90`? If you have no evidence, remove those lines.

**Comment by darko-marinov** [comment]
Please split this PR into multiple PRs, so that each is easier to review (and accept). for Guava, you can add the new test if you can check that it was `ID` in some commit between where it was moved and where it got marked as `@Ignore`. If so, the new row should have eventual status `Skipped` with the note that is the commit that added `@Ignore`.

File: pr-data.csv [hunk 1]
```diff
@@ -3020,7 +3020,7 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar
...
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testReferencedGroupWithoutDefiningReferencedAttribute,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testReferencedUserUsingReferencedAttribute,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testReferencedUserWithoutDefiningReferencedAttribute,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testReferencedUserWithoutDefiningReferencedAttribute,ID,MovedOrRenamed,,https://github.com/apache/nifi/commit/e4ec0cb20d63a38c4e8da169b0bbd26d0317fb90
```
Comment (darko-marinov) [inline pr-data.csv:3023]: The commit SHA in the note seems to be wrong.

**Comment by darko-marinov** [comment]
Please email us the commands to confirm your claim for `MovedOrRenamed`.

File: pr-data.csv (hunk 1 above)
Comment (Qi-Cui) [reply pr-data.csv:3023]: email with details sent

**Comment by darko-marinov** [comment]
Thanks for sharing all the details in your email.

**Pull request merged**